<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuscleViewer</class>
 <widget class="QWidget" name="MuscleViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>932</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MuscleViewer</string>
  </property>
  <widget class="QPushButton" name="loadFrames">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Load Frames</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>890</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Save Fit</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="frameScrollBar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>860</y>
     <width>881</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="GraphicsView" name="frameView">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>881</width>
     <height>701</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="frameNumber">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>890</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="play1X">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>890</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QPushButton" name="play2X">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>890</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playStop">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>890</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="GraphicsView" name="frameHist">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>190</y>
     <width>131</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadFit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>890</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Load Fit</string>
   </property>
  </widget>
  <widget class="GraphicsView" name="gammaPlot">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>460</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="gammaSlider">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>590</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="sliderPosition">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="commentBox">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>640</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>440</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gama</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>620</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Comments</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Histogram</string>
   </property>
  </widget>
  <widget class="GraphicsView" name="timeSeriesPlt">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>881</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="epochName">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Epoch Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newEpoch">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>New Epoch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveEpoch">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Epochs</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="epochStart">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="epochEnd">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>70</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modelselectBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>load model</string>
   </property>
  </widget>
  <widget class="QListView" name="componentsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>131</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Time series</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Image series</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Components</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyDemixing">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>890</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply Demixing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>load profile</string>
   </property>
  </widget>
  <widget class="QComboBox" name="profileselectBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="profileName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>profile name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveProfile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save profile</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="subtractBackground">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>890</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>subtract background</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
